### PR TITLE
Created Contact Form 

### PR DESCRIPTION
**Pull request by AsimKhan2019**
Created a contact form as part of the exercise today. Applied flex properties to the section elements of the form and worked on alignment and spacing. 

Integrated Form Spree to the form of the contact section, and added the necessary attributes and max length characters to handle the limit. Also, all the inputs are required fields.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -311,3 +312,64 @@ details[open] summary {
 details[open] summary::after {
   transform: rotate(90deg);
 }
+
+#contact {
```
Comment (arslanbisharat) [inline style.css:316]: - [ ] Let's not use IDs in your CSS. It's recommended to use classes instead.

File: style.css [hunk 2]
```diff
@@ -311,3 +312,64 @@ details[open] summary {
...
+  row-gap: 15px;
+  width: 100%;
+  border-radius: 100px 0 0 0;
+  color: #fff;
+  background-image: url('img/contact form shapes-mobile@2x.svg');
+  background-repeat: repeat-y;
+  background-size: 435px;
+}
+
+#contact h1 {
```
Comment (arslanbisharat) [inline style.css:331]: - [ ] Let's not use IDs in your CSS. It's recommended to use classes instead.

File: style.css [hunk 3]
```diff
@@ -311,3 +312,64 @@ details[open] summary {
...
+  color: #fff;
+  font-size: 36px;
+}
+
+#contact p {
+  font-size: 20px;
+}
+
+#contact form {
```
Comment (arslanbisharat) [inline style.css:340]: - [ ] Let's not use IDs in your CSS. It's recommended to use classes instead.

File: style.css [hunk 4]
```diff
@@ -311,3 +312,64 @@ details[open] summary {
...
+  background-repeat: repeat-y;
+  background-size: 435px;
+}
+
+#contact h1 {
+  color: #fff;
+  font-size: 36px;
+}
+
+#contact p {
```
Comment (arslanbisharat) [inline style.css:336]: - [ ] Let's not use IDs in your CSS. It's recommended to use classes instead.

File: style.css [hunk 5]
```diff
@@ -311,3 +312,64 @@ details[open] summary {
...
+
+#contact form {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  row-gap: 10px;
+  align-self: stretch;
+}
+
+#contact form input {
```
Comment (arslanbisharat) [inline style.css:348]: - [ ] Let's not use IDs in your CSS. It's recommended to use classes instead.

File: style.css [hunk 6]
```diff
@@ -311,3 +312,64 @@ details[open] summary {
...
+  height: 48px;
+  border: 1px solid #fff;
+  border-radius: 6px;
+  padding: 10px;
+}
+
+#contact form textarea {
```
Comment (arslanbisharat) [inline style.css:355]: - [ ] Let's not use IDs in your CSS. It's recommended to use classes instead.

File: style.css [hunk 7]
```diff
@@ -311,3 +312,64 @@ details[open] summary {
...
+  height: 110px;
+  border: 1px solid #fff;
+  border-radius: 6px;
+  padding: 10px;
+  text-align: left;
+}
+
+#contact form .primary-btn {
```
Comment (arslanbisharat) [inline style.css:363]: - [ ] Let's not use IDs in your CSS. It's recommended to use classes instead.

File: style.css [hunk 8]
```diff
@@ -311,3 +312,64 @@ details[open] summary {
...
+  margin-top: 0;
+}
+
+#contact form .primary-btn:hover {
+  background-color: #fff;
+  color: #6070ff;
+}
+
```
Comment (arslanbisharat) [inline style.css:371]: - [ ] Let's not use IDs in your CSS. It's recommended to use classes instead.

File: style.css [hunk 9]
```diff
@@ -311,3 +312,64 @@ details[open] summary {
...
+#contact form .primary-btn:active {
```
Comment (arslanbisharat) [inline style.css:372]: - [ ] Let's not use IDs in your CSS. It's recommended to use classes instead.

File: index.html [hunk 10]
```diff
@@ -192,6 +192,25 @@ <h1>About<br />Myself</h1>
                 </details>
             </div> <!-- details -->
         </section> <!-- about -->
+        <section id="contact">
+            <h1>
+                Contact me
```
Comment (arslanbisharat) [inline index.html:197]: - [ ] You have commented code at `13`, `18`, `21`, and `41`. Please remove it.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @AsimKhan2019, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- [x] Pull Request is descriptive. 
- [ ] Readme is well written. 
- [ ] All technical aspects of the project has been completed. 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (AsimKhan2019) [reply style.css]: @arslanbisharat Thank you for the review. Kindly please review the updated code.

File: style.css (hunk 2 above)
Comment (AsimKhan2019) [reply style.css]: @arslanbisharat Thank you for the review. Kindly please review the updated code.

File: style.css (hunk 3 above)
Comment (AsimKhan2019) [reply style.css]: @arslanbisharat Thank you for the review. Kindly please review the updated code.

File: style.css (hunk 4 above)
Comment (AsimKhan2019) [reply style.css]: @arslanbisharat Thank you for the review. Kindly please review the updated code.

File: style.css (hunk 6 above)
Comment (AsimKhan2019) [reply style.css]: @arslanbisharat Thank you for the review. Kindly please review the updated code.

File: style.css (hunk 7 above)
Comment (AsimKhan2019) [reply style.css]: @arslanbisharat Thank you for the review. Kindly please review the updated code.

File: style.css (hunk 8 above)
Comment (AsimKhan2019) [reply style.css]: @arslanbisharat Thank you for the review. Kindly please review the updated code.

File: style.css (hunk 9 above)
Comment (AsimKhan2019) [reply style.css]: @arslanbisharat Thank you for the review. Kindly please review the updated code.

File: index.html (hunk 10 above)
Comment (AsimKhan2019) [reply index.html:197]: @arslanbisharat Thank you for the review. Kindly please review the updated code.

File: style.css [hunk 11]
```diff
@@ -311,3 +317,59 @@ details[open] summary {
 details[open] summary::after {
   transform: rotate(90deg);
 }
+
+.contact {
```
Comment (AsimKhan2019) [inline style.css:321]: - [ ] Changed #contact ID to .contact class.

**Review by AsimKhan2019 — COMMENTED** [review]
Changed the #contact ID in the style.css file to .contact Class. As advised by fellow Micronaut.

**Comment by AsimKhan2019** [comment]
@KayLemba Thank you for the feedback

**Pull request merged**